tests.py

```diff
@@ -52,4 +65,3 @@ def test_data_ingested(self):
         print('PASS!')
     else:
         print('FAIL!')
-        exit(1)
```

Comment:
Definitely need this, maybe its worth moving simplehttpserver start & cleanup into python